bloomfilter/cbits/lookup3.h

```diff
@@ -127,8 +127,16 @@ and these came close:
 
 uint32_t _jenkins_hashword(const uint32_t *k, size_t length, uint32_t initval);
 
+uint32_t _jenkins_hashwordOff(const uint32_t *k, size_t off, size_t length, uint32_t initval) {
+    return _jenkins_hashword(k + off, length, initval);
+}
+
```

Comment:
Slightly odd that these are defined in the .h file rather than declared in the header and defined in the .c file. It'll work, but only because only one .c file includes this header. The other ones defined in the header are (conditionally) `static inline`.

Reply:
If you look into that file there are other functions defined there as well.